deepaas/cmd/execute.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python -*- coding: utf-8 -*-
+
+# Copyright 2020 Spanish National Research Council (CSIC)
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import mimetypes
+import os
+import shutil
+import sys
+
+import magic
```

Comment:
Last review, I promise:

can we convert this to

    try:
        import magic
    except ImportError:
        (print error, sys.exit(1))